Sample/SampleForageSDK/Sections/CardNumber/CardNumberView.swift

```diff
@@ -45,12 +45,52 @@ class CardNumberView: UIView {
         return tf
     }()
     
-    private let statusLabel: UILabel = {
+//    private let statusLabel: UILabel = {
+//        let label = UILabel()
+//        label.text = "Enter Card Number"
+//        label.textColor = .gray
+//        label.font = UIFont.systemFont(ofSize: 14, weight: .bold)
+//        label.accessibilityIdentifier = "lbl_status"
+//        label.isAccessibilityElement = true
+//        return label
+//    }()
```

Comment:
can we remove this pls?

Reply:
[1b308b2](https://github.com/teamforage/forage-ios-sdk/pull/89/commits/1b308b2e3871d7b9864b3b4b61a85ef3d5157b67)